layer-atlas/src/main/java/com/layer/atlas/adapters/AtlasConversationsAdapter.java

```diff
@@ -245,7 +246,7 @@ public void onBindViewHolder(final ViewHolder viewHolder, int position) {
         mIdentityEventListener.addIdentityPosition(position, identities);
 
 
-        viewHolder.mTitleView.setText("");
+        viewHolder.mTitleView.setText(ConversationFormatter.getConversationMetadataTitle(conversation));
```

Comment:
what about compositeDisposable.add(chatParticipantProvider
                 .getParticipants(Util.getIdsFromIdentities(identities))
                 .subscribe(new Consumer<List<Participant>>() {
                     @Override
                     public void accept(List<Participant> participants) throws Exception {
                         if (!participants.isEmpty()) {
                             setPresenceStatus(participants, identities);
                             viewHolder.mTitleView.setText(ConversationFormatter.getConversationTitle(participants, conversation));
                             if (participants.size() > 1) {
                                 sortAvatars(participants, conversation, viewHolder);
                             } else {
                                 viewHolder.mAvatarCluster.setParticipants(new LinkedHashSet<>(participants));
                             }
                         }
                     }
                 }, new Consumer<Throwable>() {
                     @Override
                     public void accept(Throwable throwable) throws Exception {
                         Log.e("Error during participants downloading", throwable);
                     }
                 })); ???

maybe there should be check if metadata != is empty and if yes then execute code above and if not then use title and not execute this method?